gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ownership/acceptOwnership.ts

```diff
@@ -0,0 +1,54 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
+import { PublicKey } from '@solana/web3.js'
+import { CONTRACT_LIST, getContract } from '../../../lib/contracts'
+import { SolanaConstructor } from '../../../lib/types'
+
+export const makeAcceptOwnershipCommand = (contractId: CONTRACT_LIST): SolanaConstructor => {
+  return class AcceptOwnership extends SolanaCommand {
+    static id = `${contractId}:accept_ownership`
+    static category = contractId
+
+    static examples = [`yarn gauntlet ${contractId}:accept_ownership --network=devnet --state=[PROGRAM_STATE]`]
+
+    constructor(flags, args) {
+      super(flags, args)
+
+      this.require(!!this.flags.state, 'Please provide flags with "state"')
+    }
+
+    execute = async () => {
+      const contract = getContract(contractId, '')
+      const address = contract.programId.toString()
+      const program = this.loadProgram(contract.idl, address)
+
+      const owner = this.wallet.payer
+
+      const state = new PublicKey(this.flags.state)
+
+      await prompt(`Accepting ownership of ${contractId} state (${state.toString()}). Continue?`)
+
+      const tx = await program.rpc.acceptOwnership({
+        accounts: {
+          // Store contract expects an store account instead of a state acc
+          ...(contractId === CONTRACT_LIST.STORE && { store: state }),
```

Comment:
@archseer Seems like the store contract expects a different account name than the rest. Should we have same interface on every program?
(https://github.com/smartcontractkit/chainlink-solana/blob/develop/contracts/programs/store/src/lib.rs#L133)

Reply:
Yeah we could do that, I kept the name different initially since there were some calls where both `store` and ocr2 `state` had to be passed but in that case it's probably better to use `store_state`/`ocr2_state`.

Reply:
https://github.com/smartcontractkit/chainlink-solana/issues/113